py-polars/polars/expr/expr.py

```diff
@@ -3733,14 +3737,14 @@ def apply(
             This is more expensive.
         strategy : {'thread_local', 'threading'}
             This functionality is in `alpha` stage. This may be removed
-            /changed without it being considered a breaking change.
+            / changed without it being considered a breaking change.
 
             - 'thread_local': run the python function on a single thread.
             - 'threading': run the python function on separate threads. Use with
-                        care as this can slow performance. This might only speed up
-                        your code if the amount of work per element is significant
-                        and the python function releases the GIL (e.g. via calling
-                        a c function)
+              care as this can slow performance. This might only speed up
+              your code if the amount of work per element is significant
+              and the python function releases the GIL (e.g. via calling
+              a c function)
```

Comment:
this isn't showing up correctly in the[ released docs](https://pola-rs.github.io/polars/py-polars/html/reference/expressions/api/polars.Expr.apply.html)

here, they look like this:

![image](https://github.com/pola-rs/polars/assets/33491632/0a6ac780-e4f9-4c45-8476-feab33d2d210)